internal/command/scale/show_machines.go

```diff
@@ -37,10 +39,56 @@ func runMachinesScaleShow(ctx context.Context) error {
 	groupNames := lo.Keys(machineGroups)
 	slices.Sort(groupNames)
 
+	// TODO: Each machine can technically have a different Guest configuration.
+	// It's impractical to show the guest for each machine, but arbitrarily
+	// picking the first one is not ideal either.
+	representativeGuests := lo.MapValues(machineGroups, func(machines []*api.Machine, _ string) *api.MachineGuest {
+		if len(machines) == 0 {
+			return nil
+		}
+		return machines[0].Config.Guest
+	})
+
+	if flag.GetBool(ctx, "json") {
+		type groupData struct {
+			Process string
+			Count   int
+			CpuKind string
+			Cpus    int
```

Comment:
This is tiny and totally optional, but I'd suggest capitalizing the acronyms, simply because it seems to match the `--json` output style for other commands.

Reply:
They probably need to add some json tags on those fields

Reply:
+1 to both. Go convention is to capitalize acronyms like URL, ID, HTTP, ...